docs/proposals/security-context-constraints.md

```diff
@@ -1,7 +1,8 @@
 ## Abstract
 
 PodSecurityPolicy allows cluster administrators to control the creation and validation of a security
-context for a pod and containers.
+context for a pod and containers. The intent of PodSecurityPolicy is to protect the cluster from the
+pod and containers, not to protect a pod or containers from a user.
```

Comment:
This is a little bit up to interpretation as PSP should eventually factor in to who can exec into a container and therefore protect a container from a user. But that is ultimately so that the user does not compromise the cluster so I think it's correct :smile: